src/Component/Configurator/ConfigurationManager.php

```diff
@@ -292,6 +305,11 @@ public function createWriter(array $configuration): ItemWriterInterface
         $factory = $this->factory->getFactory('writer');
         $writer = $factory->createFromConfiguration($configuration, $this->getWorkFileManager());
 
+        if (isset($configuration['converter'])) {
+            $converter = $this->createConverter($configuration['converter']);
+            $this->config->getPipeline()->line(new RevertPipe($converter));
```

Comment:
$converter variable needed? $this->config->getPipeline()->line(new RevertPipe($this->createConverter($configuration['converter'])));